Apps/Immunization/src/Models/Immunization.cs

```diff
@@ -0,0 +1,50 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Immunization.Models
+{
+    using System;
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// Represents a row in the Immunization.
+    /// </summary>
+    public class Immunization
```

Comment:
new Immunization json structure is available on https://phsahealthgatewayapi-dev.azurewebsites.net/swagger/index.html > under /api/v1/Immunizations section

Reply:
The name Immunization as a class will give namespace collisions

Reply:
How about ImmunizationDTO or PhsaImmunization? or do you have any suggestion for this?

Reply:
Per our discussion today, we will add ..Model suffix to these models to avoid naming conflict with the namespaces